flutter/example/pubspec.yaml

```diff
@@ -25,6 +25,8 @@ dev_dependencies:
   integration_test:
     sdk: flutter
   pedantic: ^1.9.0
+  flutter_gherkin: 3.0.0-rc.9
+  json_serializable: ^6.3.1
```

Comment:
@rogurotus why do we need this in `example/`?

Reply:
Because medea-jason is a plugin, tests need context. We use `example/` context so it has these dependencies.